webpack_loader/config.py

```diff
@@ -9,20 +9,22 @@
 DEFAULT_CONFIG = {
     'DEFAULT': {
         'CACHE': not settings.DEBUG,
+        'CACHE_TTL': -1,
         'BUNDLE_DIR_NAME': 'webpack_bundles/',
         'STATS_FILE': 'webpack-stats.json',
         # FIXME: Explore usage of fsnotify
         'POLL_INTERVAL': 0.1,
         'TIMEOUT': None,
-        'IGNORE': ['.+\.hot-update.js', '.+\.map']
-    }
+        'IGNORE': ['.+\.hot-update.js', '.+\.map'],
+        'ASSETS_LOADER_FUNCTION': 'webpack_loader.utils.load_assets_from_filesystem',
+    },
 }
 
-user_config = getattr(settings, 'WEBPACK_LOADER', DEFAULT_CONFIG)
+user_specified_entries = getattr(settings, 'WEBPACK_LOADER', {'DEFAULT': {}})
```

Comment:
If we're hoping to merge back into upstream is there any reason to change the defaults and variable names here?

Reply:
Having the object update over-top of itself was quite confusing, and I'm pretty sure this doesn't change the behaviour.
I'm happy to change back if it'll ease acceptance.

Reply:
I vote change it back. No need to mess with that, not relevant to the PR.